javalib/src/main/scala/java/lang/Thread.scala

```diff
@@ -47,4 +47,23 @@ object Thread {
     currentThread.interruptedState = false
     ret
   }
+
+  def sleep(millis: scala.Long, nanos: scala.Int): Unit = {
+    import scala.scalanative.native._
+
+    def checkErrno() =
+      if (errno.errno == errno.EINTR)
+        throw new InterruptedException("Sleep was interrupted")
+
+    if (millis < 0) throw new IllegalArgumentException("millis must be >= 0")
+    else if (nanos < 0 || nanos > 999999)
+      throw new IllegalArgumentException("nanos value out of range")
+
+    val secs  = millis / 1000
+    val usecs = (millis % 1000) * 1000 + nanos
```

Comment:
its micros, and you are even converting `(millis % 1000) * 1000` to micros there